### PR TITLE
Adding Olive Green as a color

### PR DESCRIPTION
**Pull request by RyanDelGizzi**
I added Olive Green RAL 6003 to the abb common colors list.  I don't know how common it is, but it is able to be ordered from ABB.

### CODE REVIEW

**Review by Levi-Armstrong — DISMISSED** [review]

File: abb_resources/urdf/common_colours.xacro [hunk 1]
```diff
@@ -38,6 +38,9 @@
   <!-- RAL 6002 leaf green -->
   <xacro:property name="colour_abb_leaf_green" value="0.1764706 0.3529412 0.1529412 1" />
 
+  <!-- RAL 6003 olive green -->
+  <xacro:property name="colour_abb_olive_green" value="0.3137254 0.3254902 0.2352941 1" />
```
Comment (gavanderhoorn) [inline abb_resources/urdf/common_colours.xacro]: Could you please change this to the following style (ignore the actual values, it's a copy-paste):

```xml
value="${ 61/255} ${ 61/255} ${ 61/255} 1.0"
```

Using `xacro`'s built-in math support makes sure we use the integer values from the RGB triplet while also using maximum precision available in the conversion.

I know the other `property`s don't do this, but they should be converted.

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
Thanks for the PR :+1:

Can be merged after the minor change.

File: abb_resources/urdf/common_colours.xacro (hunk 1 above)
Comment (RyanDelGizzi) [reply abb_resources/urdf/common_colours.xacro]: Updated!

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Thanks @RyanDelGizzi, much appreciated :+1:

**Pull request merged**